ob-watcher.py: Redirect back to `/` after `/refreshorderbook` and `/rotateOb`

kristapsk opened the pull request:
Fixes #1684.

MarnixCroes reviewed (commented):
tested 1d428a6d1b4ced052fccb9dc93660b1e9ef999a5
the problem is fixed

changing the unit or percentage (after refresh) can also update the orderbook (sometimes), is that ok?

 
[orderbook2.webm](https://github.com/JoinMarket-Org/joinmarket-clientserver/assets/93143998/fa8f70f6-2574-4a81-b46f-22a8dd741c72)

kristapsk commented:
> changing the unit or percentage (after refresh) can also update the orderbook (sometimes), is that ok?

Yes, same will happen if you will just refresh the page. What `/refreshorderbook` does is that it deletes all already gathered information from in-memory sqlite database and starts from scratch. But new orders are added and old removed when makers join and leave independently from that.

kristapsk commented:
Need to test does this work when there is reverse proxy in front and actual URL base is not `/`, but something else, like with https://nixbitcoin.org/orderbook/.

kristapsk commented:
Should do the same with `/rotateOb`.

kristapsk commented:
Addressed my last two comments about handling proxied situations when URI base path isn't `/` and also modified `/rotateOb` too (but current way of doing that also isn't perfect, it modifies global state, should maybe use client side cookie instead).

theborakompanioni commented:
Just for the record: That was a breaking change. Status `302 Found` instead of `200 OK` might break client behaviour (as it did in Jam - `fetch` in a browser environment does follow redirects automatically - it starting a local reverse proxy was not setup to subsequently fetch from `127.0.0.1:62601` without adaptions).
